libraries/chain/wasm_interface.cpp

```diff
@@ -219,7 +219,7 @@ namespace eosio { namespace chain {
                try {
                   Serialization::MemoryInputStream stream((const U8 *) wasm_binary, wasm_binary_size);
                   #warning TODO: restore checktime injection?
```

Comment:
should this warning still be here?

Reply:
No, I'll remove it with the instruction counting changes.